addon/components/sl-split-grid.js

```diff
@@ -0,0 +1,564 @@
+    detailName: null,
+
+    /**
+     * Indicates when the details pane is open
+     *
+     * @property {boolean} detailsOpen
+     * @default  false
+     */
+    detailsOpen: false,
+
+    /**
+     * The name of the controller/template/view to use for the filter panel
+     *
+     * @property {string} filterName
+     * @default  null
+     */
+    filterName: null,
+
+    /**
+     * Indicates when the filter pane is open
+     *
+     * @property {boolean} filterOpen
+     * @default  false
+     */
+    filterOpen: false,
+
+    /**
+     * The text to display on the filter panel toggle button
+     *
+     * @property {string} filterText
+     * @default  "Filter"
+     */
+    filterText: 'Filter',
+
+    /**
+     * The name of the template to use for the header section of the split-grid
+     *
+     * This template will be rendered in the left part of the split-grid's
+     * header, and effectively overrides the `title` property.
+     *
+     * @property {string} headerTemplate
+     * @default  null
+     */
+    headerTemplate: null,
+
+    /**
+     * When true, the split-grid is in a loading state
+     *
+     * @property {boolean} isLoading
+     * @default  false
+     */
+    isLoading: false,
+
+    /**
+     * The height of the split-grid's list pane content, in pixels
+     *
+     * @property {number} listContentHeight
+     * @default  600
+     */
+    listContentHeight: 600,
+
+    /**
+     * The name of the template to use for the footer of the list pane
+     *
+     * @property {string} listFooterTemplate
+     * @default  null
+     */
+    listFooterTemplate: null,
+
+    /**
+     * The "top" value for the table scroll to request a new page at
+     *
+     * @property {number} nextPageScrollPoint
+     * @default  0
+     */
+    nextPageScrollPoint: 0,
+
+    /**
+     * True when a next page of data has been requested, but before it has been
+     * received
+     *
+     * @property {boolean} pendingData
+     * @default  false
+     */
+    pendingData: false,
+
+    /**
+     * The title of the column that is currently being sorted
+     *
+     * @property {object} sortedColumnTitle
+     * @default  null
+     */
+    sortedColumnTitle: null,
+
+    /**
+     * The direction the currently sorted column is being sorted in
+     *
+     * Value is either "ascending" or "descending".
+     *
+     * @property {string} sortedDirection
+     * @default  null
+     */
+    sortedDirection: null,
+
+    // -------------------------------------------------------------------------
+    // Observers
+
+    /**
+     * Does cleanup for internal state when content length has changed
+     *
+     * @function handleNewContent
+     * @observes content.length
+     * @returns  {void}
+     */
+    handleNewContent: function() {
+        this.set( 'pendingData', false );
+
+        if ( !this.get( 'hasMorePages' ) ) {
+            this.disableContinuousPaging();
+        }
+    }.observes( 'content.length' ),
```

Comment:
I'd say if they're doing away with using "content" that it makes our usage of "content" more clear, as there's no conflict.